1-mandatory/2-people-i-know.js

```diff
@@ -377,23 +377,35 @@ In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
-let thirtyFiveOrOlder = [];
-
+let thirtyFiveOrOlder = friends.filter((singleFriend) => singleFriend.age > 35);
+///  commit from terminal
 /*
```

Comment:
I like the way you use the arrow function here Irianni. Your code is also very tidy.